elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -69,6 +69,13 @@ protected function load_commands() {
 	}
 
 	protected function setup_healthchecks() {
+		// Disable healthcheck job
+		if ( defined( 'DISABLE_VIP_SEARCH_HEALTHCHECKS' ) && true === DISABLE_VIP_SEARCH_HEALTHCHECKS ) {
```

Comment:
Should we just be using the filter that we added? `enable_vip_search_healthchecks`

https://github.com/Automattic/vip-go-mu-plugins/blob/2b03eec117a34fe3f76941d4bab8932688a314d1/elasticsearch/includes/classes/class-elasticsearch.php#L74

Reply:
I think using a constant that's checked earlier is a simpler solution to ensure the job is actually disabled. With the filter, we have to worry about the order and such

Reply:
> I sandboxed a site using this branch on a platform sandbox host + I set the constant to true. Did I do something wrong? 🤔

@chrean Sorry that was me doing it wrong. I was testing using `wp shell` but it any other scenario, the cron-control wouldn't have been loaded yet so it doesn't work. I've changed it to hook into the `init` action so it should work now.

Reply:
I guess it just depends on how sure we want to be that it's disabled - how critical it is to be disabled. Usually having a constant that we define somewhere on behalf of a site is useful when we _really_ need to be sure there is no other way to do something...I don't know if this case warrants having two different ways to disable the health checks? What are the use-cases?

Reply:
I think this will be fine as-is...